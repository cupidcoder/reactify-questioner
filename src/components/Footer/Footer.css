.Footer {
  color: #ffffff;
  background-color: #202C39;
  text-align: center;
  padding: 1.5rem;
}

.Footer p {
  font-family: 'Tangerine', cursive;
}
