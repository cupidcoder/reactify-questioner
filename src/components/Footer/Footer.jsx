import React from 'react';
import style from './Footer.css';

const Footer = () => (
  <footer>
    <p className={style.Footer}>Questioner</p>
  </footer>
);

export default Footer;
