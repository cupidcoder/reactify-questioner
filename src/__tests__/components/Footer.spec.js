import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer/Footer';

describe('Footer', () => {
  it('Should correctly save snapshot of the footer', () => {
    const FooterWrapper = shallow(<Footer />);
    expect(FooterWrapper).toMatchSnapshot();
  });
});
