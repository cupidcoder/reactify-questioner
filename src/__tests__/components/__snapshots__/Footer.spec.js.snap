// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should correctly save snapshot of the footer 1`] = `ShallowWrapper {}`;
